install.sh

```diff
@@ -184,14 +184,12 @@ for package in $BOOTSTRAP_PACKAGES; do
 
 done
 
-# Create the device.json file if it doesn't exist.
-if [ ! -f "${CREW_CONFIG_PATH}/device.json" ]; then
-  cd "${CREW_CONFIG_PATH}"
-  echo_info "\nCreating new device.json."
-  jq --arg key0 'architecture' --arg value0 "${ARCH}" \
-    --arg key1 'installed_packages' \
-    '. | .[$key0]=$value0 | .[$key1]=[]' <<<'{}' > device.json
-fi
+# Create the device.json file.
+cd "${CREW_CONFIG_PATH}"
```

Comment:
This overwrites any existing package.json file, which means that running the installer over an older install (in case something is broken) will now remove all info about existing packages. 

Is that good?

Can we pick up the existing installed packages when doing this and import into a device.json after doing some sanitizing?

We should at the minimum move package.json into a package.json.`date` if it already exists?

Reply:
The current installer already does this by clearing out `CREW_PREFIX`. There's no particular value in keeping a device.json when all packages have been deleted.